README.md

```diff
@@ -1,84 +1,68 @@
 [![Build Status](http://www.molgenis.org/jenkins/buildStatus/icon?job=molgenis)](http://www.molgenis.org/jenkins/job/molgenis/)
+
 # Welcome to MOLGENIS
 
-MOLGENIS is an collaborative open source project on a mission to generate great software infrastructure for life science research. Each app in the MOLGENIS family comes with rich data management interface and plug-in integration of analysis tools in R, Java and web services.
+MOLGENIS is an collaborative open source project on a mission to generate great software infrastructure for life science research. 
 
 The procedure below tells you how to checkout the molgenis project and build one of the example apps.
 
-## 1. Fork the molgenis repository
-
-Create a fork of the github.com/molgenis/molgenis repository for your own Github account.
-More information about forking can be found here: https://help.github.com/articles/fork-a-repo
-
-## 2. Clone your molgenis fork on your local system
-
-Pick a folder for your Git repositories, e.g.:
-
-    cd ~/git
-
-Clone your fork to this folder:
-
-    git clone https://github.com/<yourname>/molgenis.git
+## 1. Install software needed
 
-Optionally select stable molgenis version:
+install java8 SDK from http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html
 
-    git fetch --tags origin
-    git checkout -b <tag name: see https://github.com/molgenis/molgenis/releases>
-    
-## 3. Install and configure eclipse
+install eclipse (we used Luna) from https://www.eclipse.org/downloads/?osType=macosx
 
-Compiling MOLGENIS requires Java 7.
-MOLGENIS is created with help of Maven and Freemarker. You need a few eclipse plugins to work with those.
+install mysql from (we used 5.6.23 DMG) http://dev.mysql.com/downloads/mysql/
 
-Important: get the LATEST version of Eclipse and all plugins mentioned. For example, using Eclipse Juno Release 1 or lower might result in infinite build loops.
+install git from http://git-scm.com/download/mac
```

Comment:
Can we test the protocol on a windows machine and edit appropriately?

On Tue, Mar 3, 2015 at 10:37 PM, Fleur Kelpin <notifications@github.com>
wrote:

> In README.md
> <https://github.com/molgenis/molgenis/pull/2561#discussion_r25729160>:
>
> >
> > -Important: get the LATEST version of Eclipse and all plugins mentioned. For example, using Eclipse Juno Release 1 or lower might result in infinite build loops.
> > +install git from http://git-scm.com/download/mac
>
> not all users will be mac users
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/molgenis/molgenis/pull/2561/files#r25729160>.
>



-- 
Morris A. Swertz
Associate Professor Big data in biomedicine
Lead BBMRI-NL common services for IT
Head Section Systems Genetics & Genomics Coordination Center
Dept of Genetics
University Medical Center Groningen
The Netherlands